### PR TITLE
Alamofire 5

### PR DESCRIPTION
**Pull request by RomanPodymov**
Hello.
Thank you for AlamofireObjectMapper.
In this pull request I replaced Alamofire 5.0.0-rc3 with Alamofire 5.

### CODE REVIEW

**Comment by ppamorim** [comment]
@tristanhimmelman Could you please review this?

**Comment by ppamorim** [comment]
@RomanPodymov This library is abandoned. 
Anyone who wants this fix, just replace your pod with: 

```ruby
pod 'AlamofireObjectMapper', :git => 'https://github.com/T-Pro/AlamofireObjectMapper.git', :branch => 'master'
```

**Comment by ppamorim** [comment]
@RomanPodymov You forgot to update your podspec.

**Comment by RomanPodymov** [comment]
> @RomanPodymov You forgot to update your podspec.

Hello @ppamorim 
Alamofire 5.0.4 will be installed even with `s.dependency 'Alamofire', '~> 5.0.0'`.

**Comment by ppamorim** [comment]
@RomanPodymov I was getting error on my side.

**Comment by RomanPodymov** [comment]
> @RomanPodymov I was getting error on my side.

@ppamorim  What error do you get?

**Comment by ppamorim** [comment]
@RomanPodymov It was saying that it was using the dependency 5.0.0, even doing `pod repo update` and `pod update`. But now it's working after I sync with my branch and then with ours after. Probably it was the cache.

**Comment by ozgur** [comment]
Hi @tristanhimmelman 
Any update on getting this PR merged?

**Comment by akellyak47** [comment]
@ppamorim is this library confirmed abandoned by @tristanhimmelman ?

**Comment by kambala-decapitator** [comment]
@RomanPodymov Alamofire 5.1 has been released, you have to fix/update your podspec

**Comment by kambala-decapitator** [comment]
@RomanPodymov not sure why you use "\~> 5.1.0" (locking to 5.1.x) instead of "\~> 5.1"

**Comment by RomanPodymov** [comment]
> @RomanPodymov not sure why you use "~> 5.1.0" (locking to 5.1.x) instead of "~> 5.1"

I was not sure if Alamofire 5.0 and Alamofire 5.1 have the same public API.

File: AlamofireObjectMapper.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -836,7 +836,7 @@
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
-				MACOSX_DEPLOYMENT_TARGET = 10.0;
+				MACOSX_DEPLOYMENT_TARGET = 10.12;
```
Comment (RomanPodymov) [inline AlamofireObjectMapper.xcodeproj/project.pbxproj:839]: Alamofire 5.4.3 requires macOS 10.12.

**Comment by borisdipner** [comment]
SPM ready for object mapper and Codable AF5 https://github.com/borisdipner/AlamofireObjecter

**Pull request closed without merging**